scripts/check_test_inclusion.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python
+
+"""
+Verifies that all tests are a part of the project file.
+"""
+
+from __future__ import print_function
+import os
+import os.path
+import re
+import sys
+
+
+# Tests that are known not to compile in Xcode and can't be added there.
+EXCLUDED = frozenset([
+    # b/79496027
+    'Firestore/core/test/firebase/firestore/remote/serializer_test.cc',
+])
+
+
+def Main(args):
```

Comment:
None of the methods in this file have docstrings. You might consider adding them, especially since python's weakly typed. But OTOH it's a small, self-contained file, and I suspect will rarely (if ever) need to touch it.

Optional.

Reply:
Done. Thanks for the reminder. In the process of writing this I realized that this was factored badly, so reorganized.